admin/build-doc

```diff
@@ -73,6 +73,10 @@ export PYTHONPATH=$TOPDIR/src/pybind
 
 $vdir/bin/python $TOPDIR/doc/scripts/gen_mon_command_api.py > $TOPDIR/doc/api/mon_command_api.rst
 
+# empty frontend dir needed at minimum for json generation
+[ ! -d $TOPDIR/src/pybind/mgr/dashboard/frontend/dist ] && mkdir $TOPDIR/src/pybind/mgr/dashboard/frontend/dist
```

Comment:
```suggestion
mkdir -p $TOPDIR/src/pybind/mgr/dashboard/frontend/dist
```

Reply:
My question is: if that folder exists, do we risk getting an error thrown? frontend always exists, but dist does not. I wanted to make sure we never got an error if someone runs build-doc on a live cluster and the dist already exists.

Reply:
implementing this change!